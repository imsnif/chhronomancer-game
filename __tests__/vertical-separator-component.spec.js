import 'react-native'
import React from 'react'
import renderer from 'react-test-renderer'

jest.mock('mobx-react/native', () => require('mobx-react/custom'))

test('Vertical Separator => renders correctly', () => {
  const HorizontalSeparator = require('../components/vertical-separator').default
  const tree = renderer.create(
    <HorizontalSeparator length={42} />
  ).toJSON()
  expect(tree).toMatchSnapshot()
})
