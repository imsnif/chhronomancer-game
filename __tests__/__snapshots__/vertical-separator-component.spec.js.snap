// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Vertical Separator => renders correctly 1`] = `
<Text
  accessible={true}
  allowFontScaling={true}
  ellipsizeMode="tail"
  style={
    Object {
      "color": "#329932",
      "flex": 0,
      "lineHeight": 13,
      "textAlignVertical": "top",
      "width": 22.5,
    }
  }
>
   | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 
 | 

</Text>
`;
